modules/UI/videolayout/LargeVideoManager.js

```diff
@@ -412,15 +427,20 @@ export default class LargeVideoManager {
      * Shows hides the "avatar" message which is to be displayed either in
      * the middle of the screen or below the avatar image.
      *
-     * @param {null|boolean} show (optional) <tt>true</tt> to show the avatar
+     * @param {null|boolean} [show=null] <tt>true</tt> to show the avatar
```

Comment:
Thanks I will fix the docs, it should be 'undefined'.

The JS Doc syntax is for optional default value:
http://usejsdoc.org/tags-param.html